server/belga/macros/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""Here you can import/implement macros.
+
+This module will get reloaded per request to be up to date.
+
+Use `superdesk.macro_register.macros.register` for registration.
+"""
+
+import os
+import sys
+import imp
+import importlib
+
+
+def load_macros(path):
```

Comment:
I think you can just import it:
```
import belga.macros.add_prefix_to_headline
```

Reply:
I thought let's keep it consistent with other repo.